x/payment/types/square_sizes.go

```diff
@@ -0,0 +1,54 @@
+package types
+
+import (
+	"github.com/tendermint/tendermint/pkg/consts"
+)
+
+// https://github.com/celestiaorg/celestia-app/issues/236
+// https://github.com/celestiaorg/celestia-app/issues/239
```

Comment:
It's unclear why these are here 🤔

Reply:
whoops, yeah this code was originally just for test code, but those issues describe signing over all applicable square sizes
#236 #239 

This still needs a refactor and we should perform this calculation inside of `NewWirePayForMessage`.

Will do in a follow up PR later today